internal/kafka/avail_status_msg.go

```diff
@@ -25,27 +23,3 @@ func NewAvailabilityStatusMessage(msg *GenericMessage) (*AvailabilityStatusMessa
 func (m AvailabilityStatusMessage) GenericMessage(ctx context.Context) (GenericMessage, error) {
 	return genericMessage(ctx, m, m.SourceID, AvailabilityStatusRequestTopic)
 }
-
-func genericMessage(ctx context.Context, m any, key string, topic string) (GenericMessage, error) {
-	payload, err := json.Marshal(m)
-	if err != nil {
-		return GenericMessage{}, fmt.Errorf("unable to marshal message: %w", err)
-	}
-
-	// This will panic when identity was not present in the context (no error handling possible)
-	id := identity.Identity(ctx)
-
-	return GenericMessage{
-		Topic: topic,
-		Key:   []byte(key),
-		Value: payload,
-		// Keep headers written in lowercase to match sources comparison.
-		Headers: GenericHeaders(
-			"content-type", "application/json",
-			"x-rh-identity", identity.IdentityHeader(ctx),
-			"x-rh-sources-org-id", id.Identity.OrgID,
-			"x-rh-sources-account-number", id.Identity.AccountNumber,
-			"event_type", "availability_status",
-		),
-	}, nil
-}
```

Comment:
I am assuming no changes were made here.

Reply:
There is the one additional header.